src/app/src/test/kotlin/org/vaccineimpact/api/tests/clients/CeleryClientTests.kt

```diff
@@ -17,14 +21,39 @@ class CeleryClientTests : MontaguTests()
     fun `can call task`()
     {
         val client = CeleryClient()
-        val task = client.runDiagnosticReport("testGroup", "testDisease", "testTouchstone",
+        val response = client.runDiagnosticReport("testGroup", "testDisease", "testTouchstone",
                 "testScenario",
                 "test.user@example.com")
 
-        val result = task.get().mapValues { DiagnosticReportTaskResult(it.value["published"] ?: false) }
-        assertThat(result.count()).isEqualTo(1)
-        assertThat(result.entries.first().value.published).isTrue()
+        assertThat(response.statusCode).isEqualTo(200)
+        val taskId = JsonLoader.fromString(response.text)["task-id"].asText()
```

Comment:
Using flower is so much nicer, the Java celery thing would hang when getting the task result in the tests if the task failed